en/SUMMARY.md

```diff
@@ -30,6 +30,7 @@
 * [Releases](releases/README.md)
     * [Release Notes](releases/release_notes.md)
        * [v3.2 (Detailed)](releases/stable_v3.2_long.md)
+       * [v3.3 (Detailed)](releases/stable_v3.3_long.md)
```

Comment:
Might be worth having above 3.2 (reverse release order)- more people interested in latest than legacy.

Reply:
Oops. Also could use your help moving the 3.2 detailed notes into the regular docs.

Reply:
Happy to do so, but not sure of where it fits in priority. Probably not for next two weeks.

Reply:
... and we should also do the 3.3 notes at the same time ...